apps/wing/package.json

```
@@ -21,7 +21,8 @@
   },
   "scripts": {
     "build": "tsc && pnpm copy-root-readme",
-    "compile": "tsc",
+    "eslint": "eslint --ext .ts src",
+    "compile": "tsc && pnpm eslint",
```

turbo should take care of running the linter

```suggestion
    "compile": "tsc",
```

---

Ah cool 👍